lecture_notes/08 collections/1-array.cpp

```diff
@@ -1,13 +1,47 @@
-#include <stdio.h>
+//#include <stdio.h>
+#include <iostream>
+using namespace std;
 
-int main() {
-  int ages[] = { 10, 12, 15, 15, 17, 18, 18, 19, 20 };
+void printArray(int* array, int size){  
+  for (int i = 0; i < size; i++)
+  {
+    printf("%d ", array[i]);
+  }
```

Comment:
```suggestion
  for (int i = 0; i < size; i++) {
    printf("%d ", array[i]);
  }
```

Reply:
Thanks for your comment! All changes done